juriscraper/pacer/utils.py

```diff
@@ -189,6 +189,14 @@ def is_text(response):
     return False
 
 
+def is_html(response):
```

Comment:
Type hints, please. I'd also put this function into lib.html_utils.py instead of here.

Reply:
Done, added type hints, and moved to  `lib.html_utils.py`